modules/smartrtbBidAdapter.js

```diff
@@ -0,0 +1,105 @@
+import * as utils from 'src/utils';
+import {registerBidder} from 'src/adapters/bidderFactory';
+const BIDDER_CODE = 'smartrtb';
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['smrtb'],
+  isBidRequestValid: function(bid) {
+    return (bid.params.pubId !== null &&
+      bid.params.medId !== null &&
+      bid.params.zoneId !== null);
+  },
+  getDomain: function () {
+    if (!utils.inIframe()) {
+      return window.location.hostname
+    }
+    var origins = window.document.location.ancestorOrigins
+    if (origins && origins.length > 0) {
+      return origins[origins.length - 1]
+    }
+  },
+  buildRequests: function(validBidRequests, bidderRequest) {
+    let stack = (bidderRequest.refererInfo &&
+      bidderRequest.refererInfo.stack ? bidderRequest.refererInfo
+      : [])
+
+    const payload = {
+      start_time: utils.timestamp(),
+      tmax: 120,
+      language: window.navigator.userLanguage || window.navigator.language,
+      site: {
+        domain: this.getDomain(),
+        iframe: !bidderRequest.refererInfo.reachedTop,
+        url: stack && stack.length > 0 ? [stack.length - 1] : null,
+        https: (window.location.protocol === 'https:'),
+        referrer: bidderRequest.refererInfo.referer
+      },
+      imps: []
+    };
+
+    if (bidderRequest && bidderRequest.gdprConsent) {
+      payload.gdpr = {
+        applies: bidderRequest.gdprConsent.gdprApplies,
+        consent: bidderRequest.gdprConsent.consentString
+      };
+    }
+
+    for (let req of validBidRequests) {
```

Comment:
The `for...of` style is not permitted due to it not being available in IE browsers.  Can you change this around?

This should resolve the circleCI test failures.

Reply:
Thanks, fixed.